src/main/java/org/folio/service/orders/OrderRolloverService.java

```diff
@@ -54,7 +54,7 @@
 public class OrderRolloverService {
   private static final Logger logger = LogManager.getLogger();
 
-  private static final String PO_LINE_FUND_DISTR_QUERY = "fundDistribution = \"/@fundId == \"%s\"";
+  private static final String PO_LINE_FUND_DISTR_QUERY = "fundDistribution = \"/@fundId = \"%s\"";
```

Comment:
Why is there a quote before the slash ? I feel like the query should be `fundDistribution =/@fundId=...`

Reply:
It was added in scope of this PR : https://github.com/folio-org/mod-orders/pull/639/files 
Without this change this query fails

Reply:
I thought `fundId` was an array modifier for `fundDistribution` ? If this is the case we should add it to `schema.json`. If not, I am not sure I understand the syntax...

Reply:
With the right array modifier we could use `fundDistribution=/@fundId \"%s\"` and it would be much faster.
It could be a ginIndex with fieldName:fundDistribution and arrayModifiers:["fundId"].

Reply:
I have checked both query formats, existing and suggested. Both queries work fine. However mod-orders-storage logs contain warnings regarding index absense
`WARN  CQL2PgJSON           Doing FT search without index for po_line.jsonb->>'fundDistribution', CQL >>> SQL: fundDistribution = "/@fundId =  71e58b93-c469-4074-914d-45ec6621d715\"" >>> get_tsvector(f_unaccent(po_line.jsonb->>'fundDistribution')) @@ tsquery_phrase(f_unaccent('/@fundId =  71e58b93-c469-4074-914d-45ec6621d715"'))`

, an it looks like a good point for improvement in scope of separate story

Reply:
I agree, I will create a new ticket.

Reply:
[MODORDERS-869](https://issues.folio.org/browse/MODORDERS-869)